_data/settings/components/table.yml

```diff
@@ -3,54 +3,54 @@ contents:
   - name: Border color
     description: Defines a border color for table cells. A value of `default` uses either the default text or reverse text color depending on the site background color.
     var: $theme-table-border-color
-    default: "default"
+    default: '"ink"'
     type: color
   - name: Text color
```

Comment:
Should we move this `$theme-table-text-color` variable to follow the alphabetical order?

Reply:
Updated in [bee7397](https://github.com/uswds/uswds-site/pull/2016/commits/bee7397766c87f7696085a3f79e849497fe462c1)

Reply:
Alphabetized the table settings in [d1b2885](https://github.com/uswds/uswds-site/pull/2016/commits/d1b2885a58789eb3ddb1846d86cb67addad455ff).